### DISCUSSION TITLE
how to import issues and milestones from gitlab

### DISCUSSION THREAD

**Discussion by mahendaraaic** [community/community#31572] [Other Feature Feedback, Questions, & Ideas]
how to import issues and milestones from gitlab

**Comment by mochadwi** [comment]
we use this: https://github.com/piceaTech/node-gitlab-2-github for our gitlab to github migration

**Reply by DavidGriffith** [reply]
I tried using that but I keep running into API requests-per-hour limits.  There's no way to throttle that package to play nice with Github.  Trying to import a relatively small repository thus fails.
